commands.py

```diff
@@ -170,6 +143,53 @@ def _btype_command(self, *args):
         cmd.extend(args)
         return cmd
 
+class create_bucket_types(bucket_type_commands, Command):
+    """
+    Creates bucket-types appropriate for testing. By default this will create:
+
+    * `pytest-maps` with ``{"datatype":"map"}``
+    * `pytest-sets` with ``{"datatype":"set"}``
+    * `pytest-counters` with ``{"datatype":"counter"}``
+    * `pytest-consistent` with ``{"consistent":true}``
+    * `pytest-write-once` with ``{"write_once": true}``
+    * `pytest-mr`
+    * `pytest` with ``{"allow_mult":false}``
+    """
+
+    description = "create bucket-types used in integration tests"
+
+    user_options = [
+        ('riak-admin=', None, 'path to the riak-admin script')
+    ]
+
+    _props = {
+        'pytest-maps': {'datatype': 'map'},
+        'pytest-sets': {'datatype': 'set'},
+        'pytest-counters': {'datatype': 'counter'},
+        'pytest-consistent': {'consistent': True},
+        'pytest-write-once': {'write_once': True},
+        'pytest-mr': {},
+        'pytest': {'allow_mult': False}
+    }
+
+
+class setup_timeseries(bucket_type_commands, Command):
```

Comment:
I figured since TS isn't a real thing yet it would be nice to keep it separate and skippable.